Projects/XSD/V1/RiverscapesProject.xsd

```diff
@@ -137,6 +137,7 @@
       <xs:element type="SimpleFileType" name="SQLiteDB" />
       <xs:element type="SimpleFileType" name="PDF" />
       <xs:element type="SimpleFileType" name="HTMLFile" />
+      <xs:element type="SimpleFileType" name="ZipFile" />
```

Comment:
New file type

Reply:
I like this. Could be useful further down the road for either bundling new file types without the need to constantly change the XSD, or incorporating big files (as we are doing here).